searx/static/themes/simple/src/less/detail.less

```diff
@@ -68,9 +68,6 @@ article.result-images .detail {
       color: var(--color-result-detail-label-font);
       font-size: 0.9rem;
       line-height: 0.9rem;
-      white-space: nowrap;
-      text-overflow: ellipsis;
-      overflow: hidden;
```

Comment:
This code was meant to fix the overflow for image labels (like resolution, author, ...). Did you intentionally remove this?

Reply:
Opps sorry, I reverted to much / this what not intended .. its fixed now.

BTW: I had to rebase on master to solve a conflict in the files crated by `make static.build.commit`.

Reply:
:+1: